### PR TITLE
Add Git as data source

### PR DESCRIPTION
**Pull request by ryandoowan**

### CODE REVIEW

File: _data/tools.yml [hunk 1]
```diff
@@ -478,6 +478,25 @@
...
+  anchor: git
+  category: data-source
+  logo: /assets/images/logos/git.png
+  logosmall: /assets/images/logos/git-small.png
+  description: |
+    Git is a free and open source distributed version control system designed to
+    handle everything from small to very large projects with speed and
+    efficiency.
+
+    Git is easy to learn and has a tiny footprint with lightning fast
```
Comment (mosabua) [inline _data/tools.yml:491]: Remove this paragraph .. and replace with info about the connector ..

**Review by nineinchnick — APPROVED** [review]

File: _data/tools.yml [hunk 2]
```diff
@@ -478,6 +478,23 @@
...
+  anchor: git
+  category: data-source
+  logo: /assets/images/logos/git.png
+  logosmall: /assets/images/logos/git-small.png
+  description: |
+    Git is a free and open source distributed version control system designed to
+    handle everything from small to very large projects with speed and
+    efficiency.
+
+    Git has a Trino connector that allows the user to access git repository
```
Comment (martint) [inline _data/tools.yml:491]: This should be "Trino has a Git connector", or "There is a Git connector for Trino", or "Trino can read a Git repository via a connector", but saying that Git has a connector is confusing and a bit misleading.

File: _data/tools.yml [hunk 3]
```diff
@@ -478,6 +478,23 @@
...
+  anchor: git
+  category: data-source
+  logo: /assets/images/logos/git.png
+  logosmall: /assets/images/logos/git-small.png
+  description: |
+    Git is a free and open source distributed version control system designed to
+    handle everything from small to very large projects with speed and
+    efficiency.
+
+    There is a Git connector for Trino that allows the user to access git repository
```
Comment (mosabua) [inline _data/tools.yml:491]: How about being consistent with other data sources.. 

Use a git repository as a data source in Trino by configuring a catalog with the git connector.

**Review by mosabua — APPROVED** [review]
Looks good now.

**Pull request merged**